applications/FluidDynamicsApplication/custom_processes/weighted_divergence_calculation_process.h

```diff
@@ -0,0 +1,338 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Riccardo Tosi
+
+#ifndef KRATOS_WEIGHTED_DIVERGENCE_CALCULATION_PROCESS_H
+#define KRATOS_WEIGHTED_DIVERGENCE_CALCULATION_PROCESS_H
+
+
+#include "includes/define.h"
```

Comment:
Have you checked that you need all these includes?

Reply:
Removed what I don't need